### DISCUSSION TITLE
💰 Bounties!

### DISCUSSION THREAD

**Discussion by cruhl** [Stability-AI/StableStudio#29] [Announcements]
Hello everyone! 👋

We're in the process of figuring out how to do bounties. 💸

I wanted to get the discussion started on how people think these should work and if there are any particular features the community really really wants to see supported?

Stay tuned for more formal plans!

**Comment by metamas** [comment]
Any further developments on how bounties will be handled?

I think the way that [tinygrad](https://github.com/tinygrad/tinygrad) has been doing bounties might offer a straightforward approach. Essentially, there's a bounty board with description, amount, and whether or not someone is currently working on that bounty. As long as someone has an open and approved issue for that bounty that they are showing consistent/meaningful progress on, they get dibs. If they fail to show progress, then the bounty is re-opened.
